sdk/python/endpoints/batch/deploy-pipelines/training-with-components/components/prepare/prepare.yml

```diff
@@ -0,0 +1,43 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: uci_heart_prepare
+display_name: UCI Heart dataset prepare
+type: command
+description:  |-
+  **Prepare UCI Heart dataset**:
+
+  Prepares the UCI Heart datasets and outputs the transformation's parameters learnt during the preprocessing. If the transofrmations are also provided as inputs, then no parameters are learnt and the same transformations indicated are applied to the data.
```

Comment:
Same comment applies here as the other UCI heart dataset descriptions :)

Reply:
Done!